src/luascript.cpp

```diff
@@ -10825,6 +10827,29 @@ int LuaScriptInterface::luaPlayerGetIdleTime(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaSendCreatureSquare(lua_State* L)
+{
+	// player:sendCreatureSquare(creature, color)
+	Player* player = getUserdata<Player>(L, 1);
+	if (!player) {
+    	lua_pushnil(L);
+    	return 1;
+	}
+
+	Creature* creature = getUserdata<Creature>(L, 2);
+	if (!creature) {
+		reportErrorFunc(L, getErrorDesc(LUA_ERROR_CREATURE_NOT_FOUND));
+    	lua_pushboolean(L, false);
```

Comment:
use the TFS API if available to push values.
it is more than anything to maintain the consistency of the code.
`pushBoolean` instead `lua_pushboolean`